rdtools/normalization.py

```diff
@@ -602,13 +586,16 @@ def trapz_aggregate(time_series, target_frequency, max_timedelta=None):
         max_interval_nanoseconds = max_timedelta.total_seconds() * 10.0**9
```

Comment:
Now that it's a private function, do we need to allow `max_timedelta` to be None?  I think `energy_from_power` will always provide the appropriate value.

Reply:
Not quite. When I made the change it blew up a lot of tests.`max_timedelata` get's passed straight in, even if it is `None`. However this caused me to notice that `_aggregate()` and `interpolate()` have slightly different default behavior when they get `max_timedelta=None` and that is undesirable for this application. Better to be specific. So I modified `energy_from_power` to calculate and pass `max_timedelta`.